File: edudoc/src/app/model/interfaces/custom/claim-audit-request.dto.ts
import { EncounterStatuses } from '@model/enums/encounter-status.enum';

export interface IClaimAuditRequestDto {
    EncounterStudentId: number;
    StatusId: EncounterStatuses;
    ReasonForReturn: string;
    ReasonForAbandonment: string;
}

File: edudoc/src/Model/DTOs/ClaimAuditRequestDto.cs
using System.Data.SqlClient;
using System.Data.SqlClient;

namespace Model.DTOs
{
    public class ClaimAuditRequestDto
    {
        public int EncounterStudentId { get; set; }
        public int StatusId { get; set; }
        public string ReasonForReturn { get; set; }
        public string ReasonForAbandonment { get; set; }
    }
}
